README.md

```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# School Library
```

Comment:
- [ ] Kindly provide information for empty sections in your readme file to make it more professional.

![image](https://user-images.githubusercontent.com/65068771/157877471-315b3613-153c-448b-a660-9a587739402e.png)

Reply:
This issue is well resolved! ✔️